packages/@webex/plugin-meetings/src/mediaQualityMetrics/config.ts

```diff
@@ -4,6 +4,11 @@ export const emptyMqaInterval = {
   intervalMetadata: {
     peerReflexiveIP: '0.0.0.0',
     peripherals: [],
+    cpuInfo: {
+      numberOfCores: 1,
+      description: 'web client',
+      architecture: 'unknown',
```

Comment:
Not something we need to do now, but I think it might be good to have some type checking for architecture since it can only be specific values, even if we're hardcoding it as 'unknown' for now. But we'll handle that when we import the types from the event dictionary when we do the StatsAnalyzer refactor.

Reply:
At the beginning, I want to mention that we have a special TS library for types from 'event-dictionary'. As part of our types improvements, we should add this here later. I'd like to create a task specifically for this update so we don't forget. It's important to handle this separately as it may require some changes. We need to be prepared for potential issues or mismatches that may arise.